src/web100srv_unit_tests.c

```diff
@@ -200,7 +200,7 @@ void run_ssl_test(void test_fn(int port, const char *hostname)) {
   pid_t server_pid;
   int server_exit_code;
   int err;
-  char *server_args[] = {"--tls_port", tls_port_string, "--private_key", private_key_file, "--certificate", certificate_file, "--snaplog", "--log_dir", "/tmp", NULL};
+  char *server_args[] = {"--tls_port", tls_port_string, "--private_key", private_key_file, "--certificate", certificate_file, "--snaplog", "--tcpdump", "--cputime", "--multiple", "--max_clients=40", "--log_dir", "/tmp", NULL};
```

Comment:
Actually, there is a condition where the flags to the tls server are duplicated (I saw this during our testing).

In particular, these flags were duplicate (I don't know if this is the place where it's happening or not).

`--snaplog --tcpdump --cputime  --log_dir /tmp`